src/main/java/cubicchunks/world/column/Column.java

```diff
@@ -84,8 +87,8 @@ public Column(ICubeProvider provider, ICubicWorld world, int x, int z) {
 	//=================================================
 
 	/**
-	 * Returns Y position of the block directly above the top non-transparent block,
-	 * or null is there are no non-transparent blocks
+	 * Return Y position of the block directly above the top non-transparent block, or {@link Coords#NO_HEIGHT} is there
```

Comment:
I think it actually returns NO_HEIGHT+1 because it needs to add 1 to the value returned by heightmap...
Heightmap stores Y of the op block, this returns Y of the block above